src/main/java/com/depromeet/domain/missionRecord/application/MissionRecordService.java

```diff
@@ -165,6 +168,96 @@ public void deleteInProgressMissionRecord() {
         }
     }
 
+    @Transactional(readOnly = true)
+    public MissionStatisticsResponse findMissionStatistics(Long missionId) {
+        final Mission mission = findMissionById(missionId);
+        final LocalDateTime startedAt = mission.getStartedAt();
+        final LocalDateTime finishedAt = mission.getFinishedAt();
+        final LocalDateTime today = LocalDateTime.now();
+
+        List<MissionRecord> missionRecords =
+                missionRecordRepository.findAllByCompletedMission(missionId);
+
+        // 달성률
+        double totalMissionAttainRate =
+                calculateMissionAttainRate(
+                        missionRecords.size(),
+                        Duration.between(startedAt, finishedAt.isBefore(today) ? finishedAt : today)
+                                        .toDays()
+                                + DAYS_ADJUSTMENT);
+
+        // 시간표 생성
+        List<FocusMissionRecordItem> timeTable = generateRecordTimeTable(missionRecords);
+
+        // 최대 연속성 계산
+        long maxContinuousSuccessDay =
+                calculateMaxContinuousSuccessDay(startedAt, finishedAt, missionRecords);
+
+        long totalSymbolStack = 0;
+        long sumDuration = 0;
+        for (FocusMissionRecordItem timeOfDay : timeTable) {
+            totalSymbolStack += timeOfDay.symbolStack();
+            sumDuration += timeOfDay.durationMinute();
+        }
+
+        // 전체 수행 시간 (시간)
+        long totalMissionHour = sumDuration / 60;
+
+        // 전체 수행 시간 (분)
+        long totalMissionMinute = sumDuration % 60;
+
+        return MissionStatisticsResponse.of(
+                totalMissionHour,
+                totalMissionMinute,
+                totalSymbolStack,
+                maxContinuousSuccessDay,
+                missionRecords.size(),
+                totalMissionAttainRate,
+                startedAt,
+                finishedAt,
+                timeTable);
+    }
+
+    private List<FocusMissionRecordItem> generateRecordTimeTable(
+            List<MissionRecord> missionRecords) {
+        return missionRecords.stream()
+                .map(
+                        record ->
+                                FocusMissionRecordItem.of(
+                                        record.getDuration().toMinutes() / 10,
+                                        record.getDuration().toMinutes(),
+                                        record.getStartedAt(),
+                                        record.getFinishedAt()))
```

Comment:
```suggestion
                .map(FocusMissionRecordItem::of)
```

미션기록을 인자로 받아서 메서드 레퍼런스로 개선할 수 있을 것 같습니다.